indico/modules/rb_new/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -344,6 +336,17 @@ class RoomEditModal extends React.Component {
         );
     };
 
+    renderEquipmentList = ({input, ...props}) => {
+        let label = Translate.string('Add new equipment');
```

Comment:
can be const, or simply defined inside the prop below

Reply:
Actually, I have a problem with this. Label prop doesn't seem to be passed down.

Reply:
You need to use `componentLabel` for this. `label` creates a label before the component. See the ReduxFormField code ;)